tests/test_validate.py

```diff
@@ -63,9 +63,40 @@ def test_validate_yaml():
         },
     )
     assert errors == [
-        " - test.yaml (root.0): 8 is not of type 'object' (rule: properties.root.items.type)",
-        " - test.yaml (root.1.test): 8 is not of type 'string' (rule: properties.root.items.properties.test.type)",
-        " - test.yaml (root.2): 'test' is a required property (rule: properties.root.items.required)",
+        "-- test.yaml root.0: 8 is not of type 'object'",
+        "-- test.yaml root.1.test: 8 is not of type 'string'",
+        "-- test.yaml root.2: 'test' is a required property",
+    ]
+
+
+def test_validate_deep():
+    errors, data = validate(
+        "test.yaml",
+        yaml.load(
+            """
+root:
+  level2:
+    test: 8"""
+        ),
+        {
+            "type": "object",
+            "properties": {
+                "root": {
+                    "type": "object",
+                    "properties": {
+                        "level2": {
+                            "type": "object",
+                            "properties": {
+                                "test": {"type": "string"},
+                            },
+                        }
+                    },
+                },
+            },
+        },
+    )
```

Comment:
strange indentation no ?

Reply:
For me, it's correct, It's done by Black...